src/App.js

```diff
@@ -1,26 +1,20 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
```

Comment:
You should decide on using double or single quotes. My suggestion would be using double quotes.